tools/docs/github-query-items.ts

```diff
@@ -59,18 +60,17 @@ export async function getOpenGitHubItems(): Promise<RenovateOpenItems> {
     return result;
   }
 
-  const q = `repo:renovatebot/renovate type:issue is:open`;
-  const per_page = 100;
+  if (!process.env.GITHUB_TOKEN) {
+    logger.warn(
```

Comment:
throw if `process.env.CI` so we don't build without on main

Reply:
Is this a new logic, or did it get dropped during this change somehow?